### DISCUSSION TITLE
Can this interface with oobabooga?

### DISCUSSION THREAD

**Discussion by LiamPerson** [BerriAI/litellm#543] [Q&A] [answered]
I am wanting to interface with ChatDev and there is an active pull request to support this project as an interface. My LLM is hosted via [oobabooga's web ui](https://github.com/oobabooga/text-generation-webui) and so I am wondering if I can use this tool to interface oobabooga's API with ChatDev.

**Comment by ishaan-jaff** [comment, marked as answer]
Yes we support it @YeloPartyHat 

Here's how to use it with LiteLLM: https://docs.litellm.ai/docs/tutorials/oobabooga

**Comment by ishaan-jaff** [comment]
we still need to add support for exception mapping + add to docs cc @krrishdholakia

**Reply by ghost** [reply]
@YeloPartyHat oobabooga also has an openai compatible API - https://github.com/oobabooga/text-generation-webui/tree/main/extensions/openai
